src/Employment/WorkContractRequest_v1.0.py

```diff
@@ -0,0 +1,340 @@
+import datetime
+from datetime import date
+from enum import Enum
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class WorkContractRequest(CamelCaseModel):
+    end_user_authentication_token: str = Field(
+        ..., description="The end user authentication token"
+    )
+
+
+class EmployerInfo(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        max_length=250,
+        description="The official name of the employer",
+        examples=["Staffpoint Oy"],
+    )
+    business_id: str = Field(
+        ...,
+        title="Business ID",
+        max_length=250,
+        description="The business ID of the employer",
+        examples=["2492090-1"],
+    )
+    street_name: str = Field(
+        ...,
+        title="Street Name",
+        max_length=10,
+        description="The street name of the employer contact address",
+        examples=["Ruoholahdenkatu 17 A 6"],
+    )
+    postal_code: str = Field(
+        ...,
+        title="Postal Code",
+        max_length=250,
+        description="The postal code of the employer address",
+        examples=["00180"],
+    )
```

Comment:
Do we really want `Postal code` to be 250 characters and `Street name` to be 10?

Reply:
I believe the example doesn't fit in "street name" validations